drivers/timer/stm32_lptim_timer.c

```diff
@@ -129,7 +129,8 @@ int sys_clock_driver_init(const struct device *dev)
 	irq_enable(DT_IRQN(DT_NODELABEL(lptim1)));
 
 #ifdef CONFIG_SOC_SERIES_STM32WLX
-	__HAL_LPTIM_LPTIM1_EXTI_ENABLE_IT();
+	/* Enable the LPTIM1 wakeup EXTI line */
+	LL_EXTI_EnableIT_0_31(LL_EXTI_LINE_29);
```

Comment:
Seems there was a fixup issue. Can you move this to first commit?

Reply:
Oh yes, thanks for catching this!